web-app/src/routes/Home/Home.css

```diff
@@ -62,11 +62,11 @@
 .Home-create-marker-button span {
     text-align: center;
     height: 20px;
-    min-width: 123px;
+    width: 123px;
     color: #FFFFFF;
```

Comment:
you can use short notation here: #fff

Reply:
More importantly, this was a bad merge resolution. The min-width change was introduced to fix a bug, in 14dc65a2e1b.